setup/_common.sh

```diff
@@ -128,9 +128,16 @@ function clone_or_update_and_cd() {
     # and re-enable.
 
     if [[ ! -d $parentdir/$reponame/.git ]]; then
-      echo "Cloning new copy of $orgname/$reponame"
       CLONE_DEST=/tmp/${reponame}_$(date +"%s")
-      git clone https://github.com/$orgname/$reponame.git $CLONE_DEST
+      thetopdir="$HOME/githubalt/$reponame"
```

Comment:
E.g. if you
1) clone repo **https://github.com/milk-org/milk.git** to ***local*** directory **~/githubalt/milk/**, and
2) checkout branch **dev-no-lpacke** to that local directory,
then this script will use that local repo and branch as the source repo and branch, instead of the default branch dev from Github.

The same goes for any $orgname (milk-org, magao-x, etc.) and $reponame (milk, MagAOX, config, etc.).

This changes nothing if the corresponding repo sub-directory (milk/, MagAOX/, config/, etc.) is not under ~/githubalt/.